src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -168,6 +173,16 @@ const DetailsWidget = ({
           </div>
         </div>
       )}
+      {detailsForAction.Permissions && (
+        <div className={styles.item}>
+          <div className={styles.label}>
+            <FormattedMessage {...MSG.roles} />
+          </div>
+          <div className={styles.value}>
+            {values?.roles && <DetailsWidgetRoles roles={values.roles} />}
+          </div>
+        </div>
+      )}
```

Comment:
![Screenshot from 2021-02-05 15-52-16](https://user-images.githubusercontent.com/1193222/107045277-02a19580-67ce-11eb-9de2-d83ac8a9ae65.png)

Please add more padding to the section that displays multiple roles. In cases like this, when there's two or more roles being set, it breaks the UI